src/extensions/cookie/bootstrap-table-cookie.js

```diff
@@ -34,135 +34,42 @@ const UtilsCookie = {
 
     return searchControls
   },
-  cookieEnabled () {
+  isCookieSupportedByBrowser () {
     return !!(navigator.cookieEnabled)
```

Comment:
do we need that double negation, im sure we can remove it ?

Reply:
I agree with @UtechtDustin: https://developer.mozilla.org/en-US/docs/Web/API/Navigator/cookieEnabled